watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination_withRO.py

```diff
@@ -273,23 +299,29 @@ def set_operating_conditions(m):
     return
 
 
-def initialize_system(m):
-    prtrt = m.fs.pretreatment
+def initialize_system(m, include_pretreatment):
+    if include_pretreatment == True:
+        prtrt = m.fs.pretreatment
+    else:
+        pass
     dye_sep = m.fs.dye_separation
     desal = m.fs.desalination
 
     # initialize feed
     solve(m.fs.feed, checkpoint="solve flowsheet after initializing feed")
 
-    # pretreatment
-    propagate_state(m.fs.s_feed)
-    s = SequentialDecomposition()
-    s.options.tear_set = []
-    s.options.iterLim = 1
-    s.run(prtrt, lambda u: u.initialize())
-
-    # initialized nf
-    propagate_state(prtrt.s01)
+    # initialize pretreatment
+    if include_pretreatment == True:
+        propagate_state(m.fs.s_feed)
+        s = SequentialDecomposition()
+        s.options.tear_set = []
+        s.options.iterLim = 1
+        s.run(prtrt, lambda u: u.initialize())
```

Comment:
```suggestion
        s.run(prtrt, lambda u: u.initialize())
```
This looks like the only thing that would need to be in the conditional--i.e., whether you pass `prtrt` or `dye_sep`.